twilight-validate/src/request.rs

```diff
@@ -1115,6 +1152,23 @@ mod tests {
         assert!(communication_disabled_until(err_timestamp).is_err());
     }
 
+    #[test]
+    fn create_auto_moderation_rule_mention_total_limit_max() {
+        assert!(create_auto_moderation_rule_mention_total_limit(0).is_ok());
+        assert!(create_auto_moderation_rule_mention_total_limit(49).is_ok());
+
+        assert!(create_auto_moderation_rule_mention_total_limit(50).is_err());
```

Comment:
I think this should be
```suggestion
        assert!(create_auto_moderation_rule_mention_total_limit(50).is_ok());

        assert!(create_auto_moderation_rule_mention_total_limit(51).is_err());
```